src/appleseed/renderer/modeling/bsdf/energycompensation.cpp

```diff
@@ -129,6 +129,15 @@ AlbedoTable2D::AlbedoTable2D()
     m_avg_table = m_albedo_table + TableSize * TableSize;
 }
 
+AlbedoTable2D::AlbedoTable2D(const float* table)
```

Comment:
Why are we passing a `const float*` if we need to `const_cast<>` it away immediately after?

Reply:
The array is a const float* but to make this class work with both external tables and computed tables I need the const cast